docs/kubernetes/flannel-bigip-info.rst

```diff
@@ -3,52 +3,47 @@
 
 .. _flannel-bigip-info:
 
-BIG-IP device integration with flannel VXLAN in Kubernetes
-==========================================================
+BIG-IP and flannel VXLAN Integration
+====================================
+
+This document provides a general overview of the BIG-IP device integration with flannel VXLAN in Kubernetes. For set-up instructions, see :ref:`use-bigip-k8s-flannel`.
 
 Overview of Cluster Networking with flannel in Kubernetes
 ---------------------------------------------------------
 
-.. sidebar:: :fonticon:`fa fa-info-circle` Related
+.. sidebar:: :fonticon:`fa fa-info-circle` Related:
 
    Read about the Kubernetes `Cluster Network`_ and `Using flannel with Kubernetes`_.
 
-`Flannel`_ is a layer 3 network fabric (or, in their words, "a virtual network that attaches IP addresses to containers"). Flannel assigns a subnet to each Node and allocates an IP address within the Node's subnet to each of the Node's Pods.
-
-When you apply the `flannel manifest`_ in Kubernetes, the flannel Pod deploys on each Node in the Cluster. The Pod consists of two containers:
+`Flannel`_ is a layer 3 network fabric (or, in their words, "a virtual network that attaches IP addresses to containers"). In Kubernetes, the flannel runs as a Pod on each Node in the Cluster. The Pod consists of two containers -- :code:`install-cni` and :code:`flanneld` -- which allow flannel to provide network information to Nodes and to read information about Nodes from the Kubernetes API server.
```

Comment:
`install-cni` can probably be removed from this text because it's only used to bootstrap/setup flannel in k8s when the configuration is first applied. We'll only work with the flannel pods after that.